array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,14 @@
+/* exported dropRight */
+
+// Condition is i < array.length - count to push everything to array before arg count indicates to stop
+
+function dropRight(array, count) {
+  var newArray = [];
+  for (var i = 0; i < array.length - count; i++) {
+    if (count < array.length) {
```

Comment:
This conditional shouldn't be necessary, the code should work just fine without it, since you are stopping before the `count`